packages/core-p2p/lib/manager.js

```diff
@@ -1,198 +0,0 @@
-'use strict'
-
-const logger = require('@arkecosystem/core-container').resolvePlugin('logger')
-
-const checkDNS = require('./utils/check-dns')
-const checkNTP = require('./utils/check-ntp')
-const Monitor = require('./monitor')
-const startServer = require('./server')
-
-module.exports = class PeerManager {
-  /**
-   * @constructor
-   * @param  {Object} config
-   */
-  constructor (config) {
-    this.config = config
-    this.monitor = new Monitor(this)
-  }
-
-  /**
-   * Start P2P interface.
-   * @param {Boolean} networkStart
-   */
-  async start () {
-    await this.__checkDNSConnectivity()
-    await this.__checkNTPConnectivity()
-
-    this.api = await startServer(this, this.config)
-
-    await this.monitor.start(this.config.networkStart)
-
-    return this
-  }
-
-  /**
-   * Shutdown P2P interface.
-   */
-  async stop () {
-    return this.api.stop()
-  }
-
-  /**
-   * Update network status.
-   * @return {Promise}
-   */
-  async updateNetworkStatus () {
-    await this.monitor.updateNetworkStatus()
-  }
-
-  /**
-   * Download blocks from a random peer.
-   * @param  {Number}   fromBlockHeight
-   * @return {Object[]}
-   */
-  async downloadBlocks (fromBlockHeight) {
```

Comment:
This needs moving to `monitor.js` as it catches an exception in getRandomPeers, or at least it used to. Related to this Exception spam: https://github.com/ArkEcosystem/core/issues/852

Reply:
Will check that later, should have been fine as it handles the exception itself already 🤔

Reply:
The `getRandomDownloadBlocksPeer` was throwing the exception so I moved it inside the `try catch` block in `downloadBlocks`.

Reply:
Will test now, thanks!